python/ray/experimental/raysort/main.py

```diff
@@ -30,25 +34,43 @@ def get_args():
     )
     parser.add_argument(
         "--total_data_size",
-        default=1_000_000_000,
+        default=512 * 1000 * 1024 * 1024,
         type=ByteCount,
         help="partition size in bytes",
     )
     parser.add_argument(
         "--num_mappers",
-        default=4,
+        default=128,
         type=int,
         help="number of map tasks",
     )
     parser.add_argument(
         "--num_reducers",
+        default=16,
+        type=int,
+        help="number of reduce actors",
+    )
+    parser.add_argument(
+        "--merge_concurrency",
         default=4,
         type=int,
-        help="number of reduce tasks",
+        help="max number of merge tasks per worker node",
+    )
+    parser.add_argument(
+        "--merge_factor",
+        default=16,
+        type=int,
+        help="number of mapper results to buffer before merging",
+    )
+    parser.add_argument(
+        "--reducer_input_chunk",
+        default=100 * 1024 * 1024,
+        type=ByteCount,
+        help="number of bytes to read in memory from each mapper block",
```

Comment:
Clarify that this is for the final-stage reduce, to reduce total memory usage at the reducer.